modules/services/podman-linux/containers.nix

```diff
@@ -223,55 +182,67 @@ in let
       };
 
       environmentFile = mkOption {
-        type = with types; nullOr str;
-        default = null;
+        type = with types; either str (listOf str);
+        default = [ ];
         description =
-          "Path to a file containing environment variables to set in the container.";
+          "Paths to files containing container environment variables.";
         example = literalMD ''
           `environmentFile = "/etc/environment";`
+          or
+          `environmentFile = [ "/etc/environment" "/etc/other-env" ];`
         '';
       };
 
-      ports = mkOption {
-        type = with types; listOf str;
-        default = [ ];
-        description = "A mapping of ports between host and container";
+      exec = mkOption {
+        type = with types; nullOr str;
+        description = "The command to run after the container start.";
+        default = null;
         example = literalMD ''
-          `ports = [ "8080:80" ];`
+          `exec = "sleep inf";`
         '';
       };
 
-      user = mkOption {
-        type = with types; nullOr (oneOf [ str int ]);
-        default = null;
-        description = "The user ID inside the container.";
+      extraOptions = mkOption {
```

Comment:
@bamhm182 I’m not sure about this rename from extraPodmanArgs to extraOptions. in NixOS `virtualisation.oci-containers` it is extraOptions, but I’m undecided on if this is going too far on trying to be consistent. What do you think?

Reply:
I was going for extraPodmanArgs because the option is called "PodmanArgs", whereas `virtualisation.oci-containers` tries to be agnostic between docker podman and docker. I was also concerned that extraConfig and extraOptions sound too similar.

Reply:
That said, I think it's bike shedding to think too hard on this. I'm fine with either as both make sense. Happy to leave it as extraOptions as that's how it is within NixOS oci-containers.